central/billingmetrics/csv.go

```diff
@@ -0,0 +1,34 @@
+package billingmetrics
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+var (
+	zeroTime = time.Unix(0, 0).UTC()
+)
+
+func writeCSV(metrics []storage.BillingMetrics, wio io.Writer) error {
+	w := csv.NewWriter(wio)
+	record := []string{"Timestamp", "Nodes", "Cores"}
```

Comment:
We need to be careful when we use "Cores" in customer facing output. On bare-metal systems cores and cpu/vcpu is not the same if hyper-threading is active. In general we try to avoid "cores" for this reason.

Reply:
On the contrary, we only say "CPU cores" in the code, and never vCPU.